applications/StructuralMechanicsApplication/custom_elements/truss_element_3D2N.cpp

```diff
@@ -695,12 +694,10 @@ void TrussElement3D2N::AddExplicitContribution(
             array_1d<double, msDimension>& r_nodal_inertia = GetGeometry()[i].GetValue(NODAL_INERTIA);
             int index = i * msDimension;
 
-            #pragma omp atomic
-            r_nodal_mass += mass_vector[index];
+            AtomicAdd(r_nodal_mass, mass_vector[index]);
 
             for (int k = 0; k < msDimension; ++k) {
-                #pragma omp atomic
-                r_nodal_inertia[k] += 0.0;
+                AtomicAdd(r_nodal_inertia[k], 0.0);
```

Comment:
@KlausBSautter why did you add this?

Reply:
because no nodal_inertia is added. While I am typing these words I wonder why we do not just skip this :D

Reply:
so can we remove it then?
I mean it does nothing...

Reply:
I guess so .....

Reply:
ok then @ashishdarekar please go ahead and remove it :)

Reply:
Done :)